pushmanager/templates/push-status.html

```diff
@@ -15,7 +15,7 @@ <h3 class="status-header" section="{{section}}">{{ sect_title }} <span class="it
 <h3 class="status-header" section="pickme">Pick me, pick me! <span class="item-count"></span>
 	{% if push_info['user'] == current_user or override %}<button class="message-people">msg</button>{% end %}</h3>
 <ul id="pickme-items" class="push-items push-items-section">
-	{% for request in sorted(push_contents.get('pickme', []), key=lambda x: x['created']) %}
+	{% for request in sort_pickmes(push_contents.get('pickme', []), ['urgent', 'no-verify', 'seagull']) %}
```

Comment:
Is it possible to reference Settings for test tag instead of hardcoding seagull here?